tests/unit/test_simulation.py

```diff
@@ -265,6 +268,26 @@ def test_save_load(self):
         ):
             sim.save("test.pickle")
 
+    def test_load_param(self):
+        # Test load_sim for parameters imports
+        filename = f"{uuid.uuid4()}.p"
+        save_sim = f"import pybamm; model = pybamm.lithium_ion.SPM(); params = pybamm.ParameterValues(chemistry=pybamm.parameter_sets.Chen2020); sim = pybamm.Simulation(model, parameter_values=params); sim.solve([0, 3600]); sim.save('{filename}')"  # noqa
+        subprocess.run([sys.executable, "-c", save_sim])
+
+        try:
+            pkl_obj = pybamm.load_sim(os.path.join(filename))
+        except Exception as excep:
```

Comment:
Could you use a specific pickle-related exception class instead of the general `Exception`?